javascript/URDFLoader.js

```diff
@@ -94,26 +94,33 @@ class URDFLoader {
 
         // Check if a full URI is specified before
         // prepending the package info
+
         let path = urdf;
-        if (!/^[^:]+:\/\//.test(path)) {
+
+        if( !pkg.includes(",") && (!/^[^:]+:\/\//.test(path))) {
```

Comment:
I've been using single quotes across the repo -- sorry I haven't added an eslint file, yet!

Reply:
Will update that as well :)